src/main/webapp/resources/admin/promotions.js

```
@@ -212,68 +232,175 @@ document.addEventListener('DOMContentLoaded', function() {
+                if (searchInput) {
+                    searchInput.value = '';
+                    searchInput.focus();
+                }
+            });
+        });
+    }
+
+    // 선택된 국가 하이라이트
+    function highlightCountry(dropdown, country) {
+        const countryListEl = dropdown.querySelector('[data-country-list]');
+        if (!countryListEl) return;
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)